src/app/dataScripts/httpsRequestToCallback.ts

```diff
@@ -0,0 +1,47 @@
+import https from 'node:https';
+
+export default function httpsRequestToCallback({
```

Comment:
The `httpsRequestToFile` could be updated to use this function.

Reply:
Updated as part of 4825319db98d00501351e2a72d31bd97a8ee5a1e